### PR TITLE
Publish updates from master 

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: desktop/settings/linux-settings.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+description: Docker Desktop settings
+keywords: Settings, Docker Desktop, Linux
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:3]: Perhaps we should add some keywords (include both "settings" and "preferences", to help discoverability); perhaps also add some keywords for the specific sections, as these keywords are used for search auto-complete on the homepage, which helps finding these. e.g.;

```markdown
keywords: settings, preferences, proxy, file sharing, resources, kubernetes, Docker Desktop, Linux
```

File: desktop/settings/linux-settings.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+title: Change Docker Dektop settings on Linux
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:4]: Typo: `Dektop`

File: desktop/settings/linux-settings.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- Select the Docker menu ![whale menu](../images/whale-x.png){: .inline} and then **Settings** 
+- Select the **Settings** icon from the Docker Dashboard. 
+
+## General
+
+On the **General** tab, you can configure when to start Docker and specify other settings:
+
+- **Start Docker Desktop when you log in**. This automatically starts Docker Desktop when you open your session.
+
+- **Send usage statistics**. Docker Desktop sends diagnostics, crash reports, and usage data. This information helps Docker improve and troubleshoot the application. Clear the check box to opt out.
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:20]: Looks like a lot of this content is duplicated between Linux, Mac, and Windows (but sometimes phrased _slightly_ different).

Ideally, we would either combine the pages (and call out some options to be Mac-, Windows- or Linux-specific, or move common content to a common page.

To help finding the differences, perhaps we should do a quick sync between the pages to make the common content align (that way a "compare" between the pages shows exactly what sections are different).

File: desktop/settings/linux-settings.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+- **Open Docker Desktop dashboard at startup**. Select to automatically opens the dashboard when starting Docker Desktop.
+
+- **Use Docker Compose V2**. Select to enable the `docker-compose` command to use Docker Compose V2. For more information, see [Docker Compose V2](../../compose/index.md#compose-v2-and-the-new-docker-compose-command)
+
+## Resources
+
+The **Resources** tab allows you to configure CPU, memory, disk, proxies, network, and other resources.
+
+#### Advanced
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:32]: Looks like this is a H4, but should be a H3

File: desktop/settings/linux-settings.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+- **Disk image size**. Specify the size of the disk image.
+
+- **Disk image location**. Specify the location of the Linux volume where containers and images are stored.
+
+You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you are asked if you want to use the existing image or replace it.
+
+### File sharing
+
+Use File sharing to allow local directories on the Linux host to be shared with Linux containers.
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:54]: (re: common content); if we rephrase this to (e.g.) "on your machine" (instead of "on the Linux host"), the content is the same between all platforms.

File: desktop/settings/linux-settings.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+This is especially useful for
+editing source code in an IDE on the host while running and testing the code in a container.
+By default the `/home/<user>` directory is shared. If your project is outside this directory then it must be added
+to the list. Otherwise you may get `Mounts denied` or `cannot start service` errors at runtime.
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:58]: ... and we could have a table / overview which locations are shared by default on each platform (Linux, Mac, Windows with Hyper-V, Windows with WSL2)

File: desktop/settings/linux-settings.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+>
+ * Shared folders are designed to allow application code to be edited 
+ on the host while being executed in containers. For non-code items
+ such as cache directories or databases, the performance will be much 
+ better if they are stored in the Linux VM, using a [data volume](../../storage/volumes.md)
+ (named volume) or [data container](../../storage/volumes.md).
+
+### Proxies
+
+To configure HTTP proxies, switch on the **Manual proxy configuration** setting.
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:84]: This section needs a bit of TLC; looks like there's differences between platforms (some may actually be different), but also not the same content available for all. Probably best to leave for a general rewrite of "proxies" (there's more issues in other parts of the docs).

File: desktop/settings/linux-settings.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+HOSTNAME=b7edf988b2b5
+TERM=xterm
+HOME=/root
+HTTP_PROXY=http://proxy.example.com:3128
+```
+
+For more information on configuring the Docker CLI to automatically set proxy variables for both `docker run` and `docker build`
+see [Configure the Docker client](/network/proxy#configure-the-docker-client).
```
Comment (thaJeztah) [inline desktop/settings/linux-settings.md:101]: This should probably be a relative link to the markdown file to allow IDEs to verify the link and anchor.

File: desktop/settings/mac-settings.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
+---
+description: Docker Desktop settings
+keywords: Settings, Docker Desktop, Mac
```
Comment (thaJeztah) [inline desktop/settings/mac-settings.md:3]: Same w.r.t. keywords (probably also for the windows page)

File: desktop/settings/mac-settings.md [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+title: Change Docker Dektop preferences on Mac
```
Comment (thaJeztah) [inline desktop/settings/mac-settings.md:4]: typo: Dektop

File: desktop/settings/mac-settings.md [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+---
+
+This page provides information on how to configure and manage your Docker Desktop settings.
+
+To navigate to **Preferences** either:
+
+- Select the Docker menu ![whale menu](../images/whale-x.png){: .inline} and then **Preferences** 
+- Select the **Preferences** icon from the Docker Dashboard. 
+
+### General
```
Comment (thaJeztah) [inline desktop/settings/mac-settings.md:14]: Looks like this should be a H2

File: desktop/settings/mac-settings.md [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+- **Apply & Restart** makes the directory available to containers using Docker's
+  bind mount (`-v`) feature.
+
+> Tips on shared folders, permissions, and volume mounts
+>
+ * Share only the directories that you need with the container. File sharing
+ introduces overhead as any changes to the files on the host need to be notified
+ to the Linux VM. Sharing too many files can lead to high CPU load and slow
+ filesystem performance.
+>
```
Comment (thaJeztah) [inline desktop/settings/mac-settings.md:80]: This note looks to be formatted a bit odd (just the source code, it renders fine as HTML). I think we generally;

- repeat the `>` on each line
- use a hanging indent for bullet lists to make them look more like a bullet list also when viewing the source


```markdown
> Tips on shared folders, permissions, and volume mounts
>
> * Share only the directories that you need with the container. File sharing
>   introduces overhead as any changes to the files on the host need to be notified
>   to the Linux VM. Sharing too many files can lead to high CPU load and slow
>   filesystem performance.
...
```

**Review by thaJeztah — DISMISSED** [review]
Left some comments; I was writing along when making them; let me push those suggestions as a pull request for review

(if we want to remove the `-install` suffix, it's best to make those changes first before publishing, otherwise we'll have to add redirects)

**Review by usha-mandya — APPROVED** [review]
